markymark/Classes/Extensions/NSRegularExpression+extensions.swift

```diff
@@ -7,29 +7,29 @@ import Foundation
 
 extension NSRegularExpression {
 
-    public static func expressionWithPattern(_ pattern:String) -> NSRegularExpression {
-        let options:NSRegularExpression.Options  = [.caseInsensitive]
-        return try! NSRegularExpression(pattern:pattern, options:options);
+    public static func expressionWithPattern(_ pattern: String) -> NSRegularExpression {
+        let options: NSRegularExpression.Options  = [.caseInsensitive]
```

Comment:
remove extra spacing
```suggestion
        let options: NSRegularExpression.Options = [.caseInsensitive]
```